brayns/Brayns.cpp

```diff
@@ -1066,4 +1047,4 @@ AbstractManipulator& Brayns::getCameraManipulator()
 {
     return _impl->getCameraManipulator();
 }
-}
+} // namespace brayns
```

Comment:
? different clang-format version why?

Reply:
hum... yes, I am using clang-format version 3.8.0-2ubuntu4... Which one are you using?

Reply:
hold on, this probably comes from my home PC where I am using Ubuntu 18.04, or maybe from my Mac... Let me put things back together from my Ubuntu machine.

Reply:
also 3.8 from Ubuntu 16.04. It must be a more recent version, yes.